examples/controlnet/train_controlnet.py

```diff
@@ -1049,7 +1106,11 @@ def load_model_hook(models, input_dir):
 
                 accelerator.backward(loss)
                 if accelerator.sync_gradients:
-                    params_to_clip = controlnet.parameters()
+                    if args.train_base:
+                        params_unet_decoder = (p for n, p in unet.named_parameters() if is_in_decoder(n))
+                        params_to_clip = chain(controlnet.parameters(), unet.parameters())
```

Comment:
should this be `params_to_clip = chain(controlnet.parameters(), params_unet_decoder)`? seems like otherwise `params_unet_decoder` is unused